src/dsl/expr.rs

```diff
@@ -44,19 +50,26 @@ impl Expr {
         Expr::Cast(self.clone().into(), dtype.clone())
     }
 
+    pub fn sum(&self) -> Self {
+        Expr::Agg(AggExpr::Sum(self.clone().into()))
+    }
+
     pub fn and(&self, other: &Self) -> Self {
         binary_op(Operator::And, self, other)
     }
 
     pub fn to_field(&self, schema: &Schema) -> DaftResult<Field> {
+        use AggExpr::*;
         use Expr::*;
 
         match self {
             Alias(expr, name) => Ok(Field::new(name.as_ref(), expr.get_type(schema)?)),
+            Agg(agg_expr) => match agg_expr {
+                Sum(expr) => expr.to_field(schema),
```

Comment:
I don't believe this type is correct. the ints are casted to either int64 or uint64